pipeline/stage_unittests/job.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+. /pipeline/docker-run/markers.sh
+. /pipeline/shared/duplicati.sh
+
+function get_and_extract_test_zip () {
+    travis_mark_begin "DOWNLOADING TEST DATA $CAT"
+    # test if zip file exists and contains no errors, otherwise redownload
+    unzip -t ~/download/"${CAT}"/"${testdata}" &> /dev/null || \
+    wget --progress=dot:giga "https://s3.amazonaws.com/duplicati-test-file-hosting/${testdata}" -O ~/download/"${CAT}"/"${testdata}"
```

Comment:
I believe the tests now have setup methods that download the test data as needed (see pull request #3864).  The intent was to avoid having to perform the download separately in each test environment.  If that's working correctly, is this function needed?

Reply:
I wasn't aware of that change, but yes, that means we can remove all that stuff from the pipeline script. I will do that and let's see what happens.